Examples/apps/DestinationsExample/DestinationsExample/SceneDelegate.swift

```diff
@@ -4,7 +4,6 @@
 //
 //  Created by Brandon Sneed on 5/27/21.
 //
-
```

Comment:
We like to have whitespace between header and imports, add this carriage return back in